### DISCUSSION TITLE
Cüzdan

### DISCUSSION THREAD

**Discussion by rocknstorm** [IBAX-io/go-ibax#209] [Q&A]
Arkafaşlar selam 12 harfli private key’i veşifreyi doğru yazdığım halde cüzdana erişemiyorum. Sebebi ne olabilir. Corrucpted or invalid diyor

**Comment by Sisi06** [comment]
Yeni 12 kelime talep edebilirsiniz,ben de unutmuştum
2.kez aldım.
